pump/storage/storage.go

```diff
@@ -847,15 +855,24 @@ func (a *Append) batchRequest(reqs chan *request, maxBatchNum int) chan []*reque
 }
 
 func (a *Append) writeToValueLog(reqs chan *request) chan *request {
+	ctx, cancel := context.WithCancel(context.Background())
 	done := make(chan *request, a.options.KVChanCapacity)
+	slowChaser := newSlowChaser(a.vlog, time.Second, done)
+	go slowChaser.Run(ctx)
 
 	go func() {
-		defer close(done)
+		defer func() {
+			close(done)
```

Comment:
will put into a closed chan `done` and panic in `slowChaser` because the chaser is running